internal/repository/websocket.go

```diff
@@ -162,6 +162,10 @@ func (w *WebSocket) DeleteQuestion(ctx context.Context, id uuid.UUID) (*types.Qu
 	return nil, w.sendMessageToRetro(ctx, message, nil)
 }
 
+func (s *WebSocket) GetOldRetrospectives(ctx context.Context, date time.Time) ([]uuid.UUID, error) {
+	panic("unimplemented")
```

Comment:
Does not CleanUpRetros need this to be implemented to work??

Reply:
That's just the websocket interface, actually don't matter to websocket this function, but is required because websocket must implement the repository interface